spec/PasswordPolicy.spec.js

```diff
@@ -29,7 +29,11 @@ describe("Password Policy: ", () => {
       user.set('email', 'user@parse.com');
       return user.signUp();
     }).then(user => {
-      Parse.User.requestPasswordReset("user@parse.com");
+      Parse.User.requestPasswordReset('user@parse.com').catch((err) => {
```

Comment:
@cherukumilli 
I understand that. But the point is to be able to highlight exactly which branch has caused the error and log appropriate message.